### PR TITLE
Panzer: Create first version of IOSSConnManager

### PR DESCRIPTION
**Pull request by jbcarleton**
This initial version only works for nodal field patterns.

### CODE REVIEW

File: packages/panzer/adapters-ioss/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
+TRIBITS_SUBPACKAGE(AdaptersIOSS)
+
+# Optional IOSS dependency
+##########################
+IF(${PACKAGE_NAME}_ENABLE_SEACASIoss)
+   GLOBAL_SET(PANZER_HAVE_IOSS ON)
+ELSE()
+   GLOBAL_SET(PANZER_HAVE_IOSS OFF)
+ENDIF()
+
```
Comment (rppawlo) [inline packages/panzer/adapters-ioss/CMakeLists.txt]: Do we need this block of code? This adapter code has a mandatory dependency on IOSS so I think we could eliminate this flag.

Comment (jbcarleton) [inline packages/panzer/adapters-ioss/CMakeLists.txt]: You are correct. We also don't need the if part of the if/else. I added a commit to remove it.

**Comment by jbcarleton** [comment]
General field patterns (not just nodal) are now supported.

**Comment by rppawlo** [comment]
PR has been merged. closing.

**Pull request closed without merging**